protos/Robot_Echo/SRRobot.proto

```diff
@@ -339,8 +339,8 @@ PROTO SRRobot [
               }
             ]
             endPoint Solid {
-              translation 0.19999998777297837 0.00026863610916181684 -0.00036030151878293246
-              rotation -0.9999999953968224 4.284946735166644e-05 8.585032564057752e-05 0.9344151591877564
+              translation 0.2 0 0
+              rotation -1 0 0 0
```

Comment:
The previous rotation here just rotated the wheel approximately around its axis. In fact the axis used leaves the wheels pointing fractionally inward.